pvlib/test/test_pvsystem.py

```diff
@@ -386,6 +403,35 @@ def test_calcparams_desoto(cec_module_params):
     assert_allclose(nNsVth, 0.473)
 
 
+def test_calcparams_pvsyst(pvsyst_module_params):
```

Comment:
Not critical, but you might consider a simple "API" test that passes the five computed coefficients into `singlediode` just to make sure the computation runs. This makes sure that pvsyst gets updated if the API changes.

Reply:
Fair point. I'm not going to put a new test in here because I don't clearly see how to separate issues being tested, and to avoid redundant tests. The current `test_calcparams_pvsyst` verifies correct output of `pvsystem.calcparams_pvsyst`. Adding a call to `singlediode` within `test_calcparams_pvsyst` adds a new purpose (API check) and I prefer to keep tests to a single purpose. Adding a new test that calls `calcparams_pvsyst` and then `singlediode` would duplicate the test of `PVSystem.calcparams_pvsyst` method.

Do you have an example in the `campanelli` pull request?

Reply:
No, I haven't written any tests yet and I admit that I am not familiar with this particular mocker setup. If a mock call can verify the API, then that would be great. I don't mean to hold up this PR and I'll look into it further with the Campanelli model.